assets/scss/layouts/products/_productSaleBadges.scss

```diff
@@ -0,0 +1,112 @@
+// =============================================================================
+// PRODUCT Sale Badges (CSS)
+// =============================================================================
+
+// BURST BADGE
+// -----------------------------------------------------------------------------
+.sale-flag-star,
+.sale-flag-star:before,
+.sale-flag-star:after {
+    background: stencilColor("color_badge_product_sale_badges");
+    display: block;
+    float: none;
+    height: 50px;
+    overflow: visible;
+    position: relative;
+    transform: scaleX(1) scaleY(1) scaleZ(1);
+    transform-origin: 50% 50% 0;
+    width: 50px;
+    z-index: auto;
+}
+
+.sale-flag-star:before {
+    content: "";
+    left: 0;
+    position: absolute;
+    top: 0;
+    transform: rotateZ(30deg) scaleX(1) scaleY(1) scaleZ(1);
+}
+
+.sale-flag-star:after {
+    content: "";
+    left: 0;
+    position: absolute;
+    top: 0;
+    transform: rotateZ(60deg) scaleX(1) scaleY(1) scaleZ(1);
+}
+
+.starwrap {
+    height: 50px;
+    position: absolute;
+    top: 10px;
+    width: 50px;
+}
+
+span.sale-text-burst {
+    color: stencilColor("color_text_product_sale_badges");
+    font-weight: 600;
+    line-height: 1rem;
+    position: absolute;
+    text-align: center;
+    top: 20%;
+    width: 100%;
+    z-index: 999;
+}
+
+// -----------------------------------------------------------------------------
+// TOP LEFT BADGE
+// -----------------------------------------------------------------------------
+
+.sale-flag-side {
+    background: stencilColor("color_badge_product_sale_badges");
+    border-radius: 0 50px 50px 0;
+    color: stencilColor("color_text_product_sale_badges");
+    float: left;
+    font-size: 15px;
+    font-weight: 700;
+    height: 20px;
+    line-height: 16px;
+    margin-bottom: 4px;
+    padding: 2px 10px;
+    position: absolute;
```

Comment:
To make it easier to follow our spacing rhythm we prefer people try to follow the spacing map. You can do things like `spacing("single") + spacing("quarter")` etc to get more specific values out of it. You've got tiny values - is there a reason `spacing("eighth")` won't work instead of `2px` for example?